### PR TITLE
replace outdated screenshot

### PR DESCRIPTION
**Pull request by MaratMussabekov**
https://github.com/MicrosoftDocs/microsoft-365-docs/issues/1740

### CODE REVIEW

**Comment by MaratMussabekov** [comment]
Dear @mypil, for your information

Thank you

**Comment by mypil** [comment]
@JohanFreelancer9 - kindly do the copy editor review for this PR. Thanks.

File: microsoft-365/admin/activity-reports/email-apps-usage.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ You can get a view into email apps activity by looking at the **Users** and **Cl
 |6.  <br/> | On the **Users** chart, the Y axis is the total count of unique users that connected to an app on any day of the reporting period.  <br/>  On the **Users** chart, the X axis is number of unique users that used the app for that reporting period.  <br/>  On the **Apps** chart, the Y axis is the total count of unique users who used a specific app during the reporting period.  <br/>  On the **Apps** chart, the X axis is the list of apps in your organization.  <br/>  On the **Versions** chart, the Y axis is the total count of unique users using a specific version of Outlook desktop. If the report can't resolve the version number of Outlook, the quantity will show as Undetermined.  <br/>  On the **Versions** chart, the X axis is the list of apps in your organization.  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/email-apps-usage.md]:
```suggestion
|6.  <br/> | On the **Users** chart, the Y axis is the total count of unique users that connected to an app on any day of the reporting period.  <br/>  On the **Users** chart, the X axis is number of unique users that used the app for that reporting period.  <br/>  On the **Apps** chart, the Y axis is the total count of unique users who used a specific app during the reporting period.  <br/>  On the **Apps** chart, the X axis is the list of apps in your organization.  <br/>  On the **Versions** chart, the Y axis is the total count of unique users using a specific version of Outlook desktop. If the report can't resolve the version number of Outlook, the quantity will show as **Undetermined**.  <br/>  On the **Versions** chart, the X axis is the list of apps in your organization.  <br/> |
```

Comment (JohanFreelancer9) [reply microsoft-365/admin/activity-reports/email-apps-usage.md]: Please BOLD "Undetermined" in the second last line.

File: microsoft-365/admin/activity-reports/email-apps-usage.md [hunk 2]
```diff
@@ -51,7 +51,7 @@ You can get a view into email apps activity by looking at the **Users** and **Cl
...
 |7.  <br/> |You can filter the series you see on the chart by selectingan item in the legend. For example, on the **Users** chart, select **Mac mail** or **Outlook** ![List of email clients. Select the email client to get more reporting data on that client.](../../media/19b9da1b-7b69-4a04-8527-38349f859e84.png) to see only the info related to each one. Changing this selection doesn't change the info in the grid table. Mac mail, Outlook for Mac, Outlook mobile, Outlook desktop, and Outlook on the web are examples of email apps you may have in your organization.  <br/> |
```
Comment (JohanFreelancer9) [inline microsoft-365/admin/activity-reports/email-apps-usage.md]:
```suggestion
|7.  <br/> |You can filter the series you see on the chart by selecting an item in the legend. For example, on the **Users** chart, select **Mac mail** or **Outlook** ![List of email clients. Select the email client to get more reporting data on that client.](../../media/19b9da1b-7b69-4a04-8527-38349f859e84.png) to see only the info related to each one. Changing this selection doesn't change the info in the grid table. Mac mail, Outlook for Mac, Outlook mobile, Outlook desktop, and Outlook on the web are examples of email apps you may have in your organization.  <br/> |
```

Comment (JohanFreelancer9) [reply microsoft-365/admin/activity-reports/email-apps-usage.md]: Please change "selectingan" to "selecting an" in the first line.

**Review by JohanFreelancer9 — CHANGES_REQUESTED** [review]
Copy review - please merge two additional edits.

**Review by JohanFreelancer9 — APPROVED** [review]

**Comment by mypil** [comment]
@kwekuako - PR has been copy edited and is ready for final review. This is based on issue #1740.

Thank you.

**Comment by mypil** [comment]
@Sirkkuw  - PR has been copy edited and is ready for final review. This is based on issue #1740.

Thank you.

**Pull request merged**